.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: go
+go:
+  - "1.12.x"
+
+install: echo nop
```

Comment:
Is this needed?

Reply:
Yes. Travis has default behavior to instead go deps that needs an env var set to use go modules correctly. I'd set the envvar but we end up doing what it is doing anyway in `./test`. So this both avoids the problem and makes the test run a little faster. I'll add a comment.